molten-http-client/src/main/java/com/hotels/molten/http/client/OkHttpCallFactoryFactory.java

```diff
@@ -141,4 +145,11 @@ private void addEventTracing(OkHttpClient.Builder clientBuilder, RetrofitService
             clientBuilder.eventListenerFactory(DelegatingEventListener.delegatedFactory(eventListenerFactories)::apply);
         }
     }
+
+    private List<okhttp3.Protocol> getProtocol(List<Protocol> protocol) {
```

Comment:
Please rename to `convertProtocols` or `createProtocols`, since we are creating multiple protocols.
Same for netty.

Reply:
Resolved